arbitrum-client/src/client/mod.rs

```diff
@@ -38,6 +38,8 @@ pub struct ArbitrumClientConfig {
     pub rpc_url: String,
     /// The private key of the account to use for signing transactions
     pub arb_priv_key: LocalWallet,
+    /// The interval at which to poll for event filters and pending transactions
+    pub block_polling_interval: u64,
```

Comment:
nit: maybe name this `block_polling_interval_ms` to make the unit clear, same with the constants.